assets/scripts/modules/Modal.js

```diff
@@ -0,0 +1,195 @@
+import { createFocusTrap } from 'focus-trap'
+import { module as Module } from 'modujs'
+import { $html } from '../utils/dom'
+import { CUSTOM_EVENT } from '../config'
+
+/**
+ * Generic component to display a modal.
+ *
+ */
+export default class Modal extends Module {
+    /**
+     * Creates a new Modal.
+     *
+     * @param  {object} options          - The module options.
+     * @param  {string} options.dataName - The module data attribute name.
+     * @throws {TypeError} If the class does not have an active CSS class defined.
+     */
+
+    static CLASS = {
+        EL: 'is-open',
+        HTML: 'has-modal-open',
+    }
+
+    constructor(options) {
```

Comment:
```suggestion
    static CLASS = {
        EL: 'is-open',
        HTML: 'has-modal-open',
    }

    /**
     * Creates a new Modal.
     *
     * @param {object} options          - The module options.
     * @param {string} options.dataName - The module data attribute name.
     */
    constructor(options) {
```

Misplaced block comment and no related error to the `@throws`.